templates/taxi/car_detail.html

```diff
@@ -5,8 +5,19 @@
   <h1>Drivers</h1>
   <hr>
   <ul>
-  {% for driver in car.drivers.all %}
-    <li>{{ driver.username }} ({{ driver.first_name }} {{ driver.last_name }})</li>
-  {% endfor %}
+    {% for driver in car.drivers.all %}
+      <li>{{ driver.username }} ({{ driver.first_name }} {{ driver.last_name }})</li>
+    {% endfor %}
```

Comment:
The `car.drivers.all` should be `car.drivers.all()`, as `all` is a method that needs to be called.